### DISCUSSION TITLE
Changes to the istep state in vehicle API

### DISCUSSION THREAD

**Discussion by davecarlson** [bimmerconnected/bimmer_connected#475] [Fingerprint data contributions]
### Describe the issue

BMW have changed how isteps are sent over in the latest api

### Expected behavior

This is the new format
```
"softwareVersionCurrent": {
    "puStep": {
        "month": 3,
        "year": 22
    },
    "iStep": 565,
    "seriesCluster": "S18A"
},
"softwareVersionExFactory": {
    "puStep": {
        "month": 11,
        "year": 21
    },
    "iStep": 538,
    "seriesCluster": "S18A"
}
```

### Which Home Assistant version are you using?

n/a

### What was the last working version of Home Assistant Core?

n/a

### What is your region?

Rest of world

### MyBMW website

- [X] I can still successfully login to the BMW MyBMW website and the car status is available there.
- [X] I have MyBMW enabled for my vehicle.

### Number of cars

- [ ] I have 2 or more cars linked to the MyBMW account.
- [ ] I have a Mini vehicle linked to my account.

**Comment by rikroe** [comment]
Not sure what this report is pointed at.
We're not parsing either of those values in the library, just returning the data as received from API.

**Comment by davecarlson** [comment]
Mainly informative in case you did use these variables anywhere ( eg tests )

feel free to close

**Comment by rikroe** [comment]
Ah, in that case, thanks! That would rather be a discussion next time as we try to keep issues for actual bugs only. 

Do you habe any further information how `iStep` or other values could be used to differentiate between different car capabilities?
